scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -42,18 +44,22 @@ task CollectCoverage {
     Int? mem
 
     # If no padded target file is input, then do WGS workflow
-    Boolean is_wgs = select_first([padded_targets, ""]) == ""
+    Boolean is_wgs = !defined(padded_targets)
 
     # Sample name is derived from the bam filename
     String base_filename = sub(sub(sub(bam, "gs://", ""), "[/]*.*/", ""), "\\.bam$", "")
-
+ 
+    # Output file name depending on type of coverage
+    String cov_output_name = if (is_wgs && (select_first([transform, ""])  == "RAW")) then "${base_filename}.coverage.tsv.raw_cov" else "${base_filename}.coverage.tsv"
```

Comment:
Actually, I was just letting you know of the change so you could update the example command.  But after discussing with @asmirnov239 we decided to hold off on the pcov change.